packages/datadog-plugin-hapi/test/integration-test/server.mjs

```diff
@@ -0,0 +1,39 @@
+import 'dd-trace/init.js'
+import Hapi from '@hapi/hapi'
+import getPort from 'get-port'
+import axios from 'axios'
```

Comment:
It would be more "pure" to use the built-in http library to make a request. That way you wouldn't need to add axios to the sandbox. But up to you if you want to change it since it already works.